docs-devsite/vertexai-preview.schema.md

```diff
@@ -0,0 +1,264 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# Schema class
+Parent class encompassing all Schema types, with static methods that allow building specific Schema types. This class can be converted with JSON.stringify() into a JSON string accepted by Vertex REST endpoints. (This string conversion is automatically done when calling SDK methods.)
+
+<b>Signature:</b>
+
+```typescript
+export declare abstract class Schema implements SchemaInterface 
+```
+<b>Implements:</b> [SchemaInterface](./vertexai-preview.schemainterface.md#schemainterface_interface)
+
+## Constructors
+
+|  Constructor | Modifiers | Description |
+|  --- | --- | --- |
+|  [(constructor)(schemaParams)](./vertexai-preview.schema.md#schemaconstructor) |  | Constructs a new instance of the <code>Schema</code> class |
+
+## Properties
+
+|  Property | Modifiers | Type | Description |
+|  --- | --- | --- | --- |
+|  [description](./vertexai-preview.schema.md#schemadescription) |  | string | Optional. The description of the property. |
+|  [example](./vertexai-preview.schema.md#schemaexample) |  | unknown | Optional. The example of the property. |
+|  [format](./vertexai-preview.schema.md#schemaformat) |  | string | Optional. The format of the property. Supported formats: for NUMBER type: "float", "double" for INTEGER type: "int32", "int64" for STRING type: "email", "byte", etc |
+|  [nullable](./vertexai-preview.schema.md#schemanullable) |  | boolean | Optional. Whether the property is nullable. Defaults to false. |
+|  [type](./vertexai-preview.schema.md#schematype) |  | [SchemaType](./vertexai-preview.md#schematype) | Optional. The type of the property. [SchemaType](./vertexai-preview.md#schematype)<!-- -->. |
+
+## Methods
+
+|  Method | Modifiers | Description |
+|  --- | --- | --- |
+|  [array(arrayParams)](./vertexai-preview.schema.md#schemaarray) | <code>static</code> |  |
+|  [boolean(booleanParams)](./vertexai-preview.schema.md#schemaboolean) | <code>static</code> |  |
+|  [enumString(stringParams)](./vertexai-preview.schema.md#schemaenumstring) | <code>static</code> |  |
+|  [integer(integerParams)](./vertexai-preview.schema.md#schemainteger) | <code>static</code> |  |
+|  [number(numberParams)](./vertexai-preview.schema.md#schemanumber) | <code>static</code> |  |
+|  [object(objectParams)](./vertexai-preview.schema.md#schemaobject) | <code>static</code> |  |
+|  [string(stringParams)](./vertexai-preview.schema.md#schemastring) | <code>static</code> |  |
+|  [toJSON()](./vertexai-preview.schema.md#schematojson) |  | Defines how this Schema should be serialized as JSON. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global\_Objects/JSON/stringify\#tojson\_behavior |
```

Comment:
another exposed URL that should be a hyperlink

Reply:
Fixed.

Reply:
Was it intentional to remove `toJSON()` here?  Maybe I'm missing it, but it doesn't seem to be in the newly generated file?

docs-devsite/vertexai.schema.md

Reply:
Yes, it doesn't need to be publicly exposed, it's for internal use.